README.md

```diff
@@ -1,4 +1,7 @@
 # [Syncthing](https://github.com/syncthing/syncthing) Tray
+[![GitHub license](https://img.shields.io/github/license/Martchus/syncthingtray)](https://github.com/Martchus/syncthingtray/blob/master/LICENSE)
+![GitHub release (latest by date)](https://img.shields.io/github/v/release/Martchus/syncthingtray)
```

Comment:
Clicking the release shield only shows the shield itself. It would make sense to make it point to the releases section on GitHub.

Reply:
Ah, I have no idea why I forgot to do it. Will fix it in a sec

Reply:
Did address both, are there any other issue(s)?

Reply:
No, it should be fine now.